bitcanna/chain.json

```diff
@@ -34,9 +34,9 @@
   },
   "codebase": {
     "git_repo": "https://github.com/BitCannaGlobal/bcna",
-    "recommended_version": "v4.0.1",
+    "recommended_version": "v4.0.3",
```

Comment:
Please update the versions.json file accordingly

Reply:
OK; I'm going to do it. Thanks